RecoLocalCalo/HGCalRecProducers/interface/HGCalLayerTiles.h

```diff
@@ -78,8 +78,10 @@ class HGCalLayerTilesT {
     float d2 = dim2Cell1 - dim2Cell2;
     if constexpr (std::is_same_v<WRAPPER, PhiWrapper>) {
       d2 = reco::deltaPhi(dim2Cell1, dim2Cell2);
+      d2 *= d2;
+      return std::fmaf(d1, d1, d2);
```

Comment:
I think these two lines are not needed, and removing them should give the same behaviour ?